template/cordova/lib/build.js

```diff
@@ -60,18 +60,18 @@ module.exports.run = function run (buildOptions) {
     var buildConfig = parseAndValidateArgs(buildOptions);
 
     return Q().then(function () {
-        // CB-11548 use LOCALMSBUILDBINPATH environment if defined to find MSBuild
-        // If LOCALMSBUILDBINPATH is not specified use default discovery mechanism
-        if (!process.env.LOCALMSBUILDBINPATH) {
+        // CB-11548 use VSINSTALLDIR environment if defined to find MSBuild
+        // If VSINSTALLDIR is not specified use default discovery mechanism
+        if (!process.env.VSINSTALLDIR) {
             return MSBuildTools.findAllAvailableVersions();
         }
 
-        events.emit('log', 'Found LOCALMSBUILDBINPATH environment variable. Attempting to build project using that version of MSBuild');
+        events.emit('log', 'Found VSINSTALLDIR environment variable. Attempting to build project using that version of MSBuild');
 
-        return MSBuildTools.getMSBuildToolsAt(process.env.LOCALMSBUILDBINPATH)
+        return MSBuildTools.getMSBuildToolsAt(process.env.VSINSTALLDIR + "\\MSBuild\\15.0\\Bin")
```

Comment:
I don't think we should assume `15.0` here, as that will break in future releases.